### DISCUSSION TITLE
Mount persistent storage w/ e.g. databases into process

### DISCUSSION THREAD

**Discussion by phiweger** [nextflow-io/nextflow#1947] [Q&A]
I have a sequence database (~ 500 gb) deposited in a Google bucket that is used by some search process. When I execute this workflow, it spends 2 hours transferring data into the workflow execution directory. Is there a way to just mount the database bucket using `gsutils`?

Thank you!

**Comment by pditommaso** [comment]
No, nextflow does not provision file storage or mont buckets. It could manage by Tower tho, as it's done already when using AWS batch

**Reply by phiweger** [reply]
> It could manage by Tower tho

How would I do this? Intuitively, I'd try with the prerun script field in the launch API call?
